src/esp/physics/RigidBase.h

```diff
@@ -32,20 +32,26 @@ namespace physics {
 
 class RigidBase : public esp::physics::PhysicsObjectBase {
  public:
+  /**
+   * @brief Constructor for a @ref esp::physics::RigidBase.
+   * @param rigidBodyNode Pointer to the node to be used for this rigid.
+   * @param objectID the desired ID for this rigid construct.
+   * @param resMgr a reference to @ref esp::assets::ResourceManager
+   */
   RigidBase(scene::SceneNode* rigidBodyNode,
             int objectId,
             const assets::ResourceManager& resMgr)
       : PhysicsObjectBase(rigidBodyNode, objectId, resMgr),
         visualNode_(&rigidBodyNode->createChild()) {}
 
   /**
-   * @brief Virtual destructor for a @ref RigidBase.
+   * @brief Virtual destructor for a @ref esp::physics::RigidBase.
```

Comment:
... for example here the error is because `RigidBase` itself is not documented (and fully qualifying *didn't* fix the error). 

Same is I suppose with `RigidObject` and `RigidStage` below -- no full qualification should be needed.

Reply:
Where should the doc be? Right about "class RigidBase...."?

Reply:
Right above the class, yep.